### ISSUE TITLE
Add white space or some kind of connecting chrome between the selected code tab and the code it selects

### ISSUE THREAD

**Issue by erikeckhardt** [levibotelho/LearnAngular#42]
Add a little bit of white chrome above the code window so the selected code tab appears to belong to the currently selected code pane. As it is now, when the cursor is on line 1 of a blank code window, the top line is the same color as deselected tabs, and especially in the case of there being just 2 code tabs, it appears as if the inactive/deselected tab is the current tab because it visually connects to the code window through the consistent color. I'm all for minimal interfaces, but there still needs to be an "information scent" in the UI to help people more quickly grasp it. I clicked on a tab more than once, not realizing I was *already* on the tab I wanted, because I thought the deselected one was active.

**Comment by levibotelho** [comment]
Fair point. Might be easier to just change the colour of the currently selected line in Ace.
